tests/basic.json

```diff
@@ -18,6 +18,22 @@
             "expression": "foo\n.\nbar\n.baz",
             "result": "correct"
          },
+         {
+            "expression": "foo . bar . baz",
+            "result": "correct"
+         },
+         {
+            "expression": "foo\t.\tbar\t.\tbaz",
+            "result": "correct"
+         },         
```

Comment:
Trailing whitespace here.

Reply:
Are you talking about the code or the test case?
Do you want the test expression to have a trailing `\t`?

Reply:
No, in the actual json document. ;)

<img width="213" alt="image" src="https://user-images.githubusercontent.com/2829600/71592258-78492580-2b27-11ea-8f92-60fca22e829a.png">

Reply:
Should have said: "Please can you remove the trailing space characters on this line". Sorry!

Reply:
The one time I don't use my own IDE, 20% of LOC contain errors =D